app/services/publish_finder.rb

```diff
@@ -10,14 +10,38 @@ def self.call(finder_content_item)
   end
 
   def call
+    send_to_publishing_api
+    send_to_rummager
+  end
+
+private
+
+  def send_to_publishing_api
     Whitehall.publishing_api_v2_client.put_content(
       content_id,
       finder_content_item
     )
     Whitehall.publishing_api_v2_client.publish(content_id, "major")
   end
 
-private
+  def send_to_rummager
+    index = Whitehall::SearchIndex.for(:government)
+    index.add(present_for_rummager)
+  end
+
+  def present_for_rummager
+    {
+      _id: finder_content_item.fetch("base_path"),
+      link: finder_content_item.fetch("base_path"),
+      format: "finder",
+      title: finder_content_item.fetch("title"),
+      description: finder_content_item.fetch("description", ""),
+      content_store_document_type: "finder",
+      content_id: content_id,
+      publishing_app: "whitehall",
+      rendering_app: "finder-frontend",
```

Comment:
Minor nitpick, can you get these values from `finder_content_item.fetch()` like the others?